op-node/sources/receipts_test.go

```diff
@@ -226,6 +232,12 @@ func TestEthClient_FetchReceipts(t *testing.T) {
 			providerKind: RPCKindAlchemy,
 			setup:        fallbackCase(30, AlchemyGetTransactionReceipts),
 		},
+		{
+			name:         "alchemy sticky",
+			providerKind: RPCKindAlchemy,
+			staticMethod: true,
+			setup:        fallbackCase(30, AlchemyGetTransactionReceipts, AlchemyGetTransactionReceipts),
```

Comment:
How is does this verify undoing the fallback?

Reply:
Compare it against the test case below, where it changes from AlchemyGetTransactionReceipts to EthGetBlockReceipts when the bool is not true.